app/services/analytics_events.rb

```diff
@@ -3797,15 +3792,8 @@ def user_registration_email_confirmation(
   end
 
   # Tracks when user visits enter email page
-  # @param [String] sign_in_a_b_test_bucket
-  # @param [Boolean] from_sign_in
-  def user_registration_enter_email_visit(sign_in_a_b_test_bucket:, from_sign_in:, **extra)
-    track_event(
-      'User Registration: enter email visited',
-      sign_in_a_b_test_bucket:,
-      from_sign_in:,
```

Comment:
is this a useful param to keep at all?

Reply:
(obviously we'd keep the misspelling from HTTP 😄 )